cypress.config.js

```diff
@@ -4,6 +4,11 @@ module.exports = defineConfig({
   e2e: {
     // baseUrl is set using environment variables because it differs between
     // development and CI setups.
+    defaultCommandTimeout: 10000,
```

Comment:
Consdier removing the `defaultCommandTimeout`.

If it turned outthe timeout does not make a difference then I suggest removing it again.